src/tasks/mixin/navigation_mixin.py

```
@@ -47,29 +49,61 @@ def start_tracking_and_align_target(self, target_feature_in_map, target_feature_
+            nav: 导航标识（OCR文本 / YOLO类别 / 特征名）。为 None 时禁用导航逻辑。
+            target_is_ocr (bool): target 是否使用 OCR 检测。
+            nav_is_ocr (bool): nav 是否使用 OCR 检测。
+            time_out (int): 最大运行时间（秒）。
+            pre_loop_callback (callable, optional): 每轮循环前执行的回调函数。
+            found_special_callback (callable, optional): 特殊状态检测回调，返回非 None 则中断并返回。
+            target_is_yolo (bool): target 是否使用 YOLO 检测。
+            nav_is_yolo (bool): nav 是否使用 YOLO 检测。
+            box (tuple, optional): target 检测区域（None 则使用默认区域）。
+            target_vertical_variance (float): 特征匹配时允许的垂直误差。
+            need_v (bool): 是否在导航丢失时周期性按 V 尝试重置视野。
+            max_run_time (float): 最大累计奔跑（ctrl 状态）时间（秒），控制的是奔跑/步行切换，不影响 w 前进键。
```

Ruff (0.15.18)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)